### ISSUE TITLE
`valid_ibinspectable` fails if you annotate a computed property as `@IBInspectable` [Swift 5.2]

### ISSUE THREAD

**Issue by marcelofabri** [realm/SwiftLint#3098]
May be related, but `valid_ibinspectable` also fails if you annotate a computed property as `@IBInspectable`:

```swift
extension Foo {
    @IBInspectable var bar: Bool {
        get { _bar }
        set { self._bar = newValue }
    }
}
```

_Originally posted by @rjchatfield in https://github.com/realm/SwiftLint/issues/3074#issuecomment-582682554_
